dask/dataframe/tests/test_io.py

```diff
@@ -668,3 +668,29 @@ def test_report_dtype_correction_on_csvs():
             assert False
         except ValueError as e:
             assert "'numbers': 'float64'" in str(e)
+
+
+def test_hdf_globbing():
+    pytest.importorskip('tables')
+    df = pd.DataFrame({'x': ['a', 'b', 'c', 'd'],
+                       'y': [1, 2, 3, 4]}, index=[1., 2., 3., 4.])
+    a = dd.from_pandas(df, 2)
+
+    pattern = '/tmp/*foo.h5'
```

Comment:
Suggest inlining this variable.  I found myself having to trace back from the `read_hdf` lines to find this one to figure out what was going on in each case.  Seeing the following would be more reassuring.

    df = dd.read_hdf('/tmp/*foo.h5', '/*.data')